rudder-inventory-ldap/SOURCES/rudder-inventory-ldap-postinst

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+set -e
+
+PACKAGE_NAME="rudder-inventory-ldap"
+LOG_FILE="/var/log/rudder/install/${PACKAGE_NAME}.log"
+LDAP_CONF="/opt/rudder/etc/openldap/slapd.conf"
+
+CFRUDDER_FIRST_INSTALL="$1"
+if [ -z "$1" ]
+then
+  echo "Usage: $0 <CFRUDDER_FIRST_INSTALL>"
+  echo " This should only be called from a package postinstall command"
+  exit 1
+fi
+
+echo "`date` - Starting ${PACKAGE_NAME} post installation script" >> ${LOG_FILE}
+
+# Remove pidfile and argsfile
+sed -i '/^[ \t]*pidfile/d' "${LDAP_CONF}"
+sed -i '/^[ \t]*argsfile/d' "${LDAP_CONF}"
+
+# Migration from pre-5.1, moving settings to slapd.conf
+if [ -f /etc/default/rudder-slapd ]; then
```

Comment:
will the file still be there in a postinst ?

Reply:
good question, it seems not, I'll make a copy of the file in preinst to make sure we don't miss it in case of a purge.